src/components/progress-bar/daikin-progress-bar.ts

```diff
@@ -0,0 +1,146 @@
+import {
+  colorFeedbackNegative,
+  colorFeedbackPositive,
+} from "@daikin-oss/dds-tokens/js/daikin/Light/variables.js";
+import { cva } from "class-variance-authority";
+import { LitElement, css, html, unsafeCSS } from "lit";
+import { customElement, property } from "lit/decorators.js";
+import tailwindStyles from "../../tailwind.css?inline";
+
+const cvaBar = cva(["w-full", "h-1", "overflow-hidden"], {
+  variants: {
+    variant: {
+      inprogress: [
+        "bg-daikinNeutral-200",
+        "relative",
+        "after:block",
+        "after:w-[--bar-width]",
+        "after:h-full",
+        "after:bg-daikinBlue-500",
+        "after:absolute",
+        "after:left-0",
+      ],
+      completed: ["bg-[--colorFeedbackPositive]"],
+      indeterminate: [
+        "bg-daikinNeutral-200",
+        "relative",
+        "after:block",
+        "after:w-1/2",
+        "after:h-full",
+        "after:bg-daikinBlue-500",
+        "after:absolute",
+        "after:left-0",
+        "after:animate-[progress-bar-indeterminate_1200ms_linear_infinite]",
+      ],
+      error: ["bg-[--colorFeedbackNegative]"],
+    },
+  },
+});
```

Comment:
completed and error has full width, which may not be intended.

```suggestion
const cvaBar = cva([
  "w-full",
  "h-1",
  "overflow-hidden",
  "relative",
  "after:block",
  "after:h-full",
  "after:absolute",
  "after:left-0",
], {
  variants: {
    variant: {
      inprogress: [
        "after:bg-daikinBlue-500"
        "after:w-[--bar-width]",
      ],
      completed: [
        "after:bg-[--colorFeedbackPositive]",
        "after:w-[--bar-width]",
      ],
      indeterminate: [
        "after:bg-daikinBlue-500",
        "after:w-1/2",
        "after:animate-[progress-bar-indeterminate_1200ms_linear_infinite]",
      ],
      error: [
        "bg-[--colorFeedbackNegative]",
        "after:w-[--bar-width]",
      ],
    },
  },
});
```